Added marketplaces in pack metadata (#1155)
demisto/content-docs: docs/packs/packs-format.md

| created | String | Pack creation time in ISO 8601 format - YYYY-MM-DDTHH:mm:ssZ, e.g. 2020-01-25T10:00:00Z |
| useCases | List | Use-cases implemented by the pack. |
| keywords | List | List of strings by which the pack can be found in Cortex XSOAR marketplace. |

> | marketplaces    | List | List of strings by which marketplace is the pack exist (XSOAR XSIAM). |

Reviewer: ```suggestion
| marketplaces    | List | List of marketplaces in which the pack can be found (XSOAR XSIAM). |
```
